dhctl/cmd/dhctl/commands/bootstrap/bootstrap.go

```diff
@@ -129,8 +129,14 @@ func bootstrapAdditionalNodesForCloudCluster(kubeCl *client.KubernetesClient, me
 
 func setBastionHostFromCloudProvider(host string, sshClient *ssh.Client) {
 	app.SSHBastionHost = host
-	app.SSHBastionUser = app.SSHUser
-	app.SSHBastionPort = app.SSHPort
+
+	if app.SSHBastionUser == "" {
+		app.SSHBastionUser = app.SSHUser
+	}
+
+	if app.SSHBastionPort == "" {
+		app.SSHBastionPort = app.SSHPort
+	}
```

Comment:
Why we need this code ?

Reply:
it fixes old bug

Reply:
Which bug ? Can we have issue about it ?

Reply:
The bug is that ssh-user overrides ssh-bastion-user. There is no open issue for this bug.